client/src/plugins/libparsec/definitions.d.ts

```diff
@@ -752,6 +775,9 @@ export interface LibParsecPlugin {
         recipient: UserID,
         role: RealmRole | null
     ): Promise<Result<null, ClientWorkspaceShareError>>
+    getBackendAddrType(
+        url: string
+    ): Promise<Result<BackendAddrType, string>>
```

Comment:
What's the `string` here? Shouldn't it be an error type?

Reply:
It's any url which will be parsed to know which BackendAddr is